substrate/frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -517,6 +536,53 @@ impl<T: MinerConfig> Miner<T> {
 				},
 			);
 
+			// ensures that the max backers per winner bounds are respected given the supports
```

Comment:
Note for auditors: This is among the only new bits of logic here. It makes sure a mined solution respects `MaxBackersPerWinner`.